shared/cfg/base.cfg

```diff
@@ -146,7 +146,9 @@ q35, arm64-pci, arm64-mmio:
 # chardev_host = $host_ip (127.0.0.1 by default)
 # chardev_ipv4 = yes
 # chardev_ipv6 = yes
+# chardev_path = *** 'path' for chardev, if not define, framework have default
```

Comment:
better to move comments in your code.

Reply:
configuration for some special case, so suggest to leave comments in code don't introduce it in base.cfg

Reply:
No strong idea about this, it is just a comment. remove it now. thanks!